File: Java/Simple_list/app/src/main/java/com/example/android/simple_list/MainActivity.java
package com.example.android.simple_list;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private TextView text_list;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        text_list = findViewById(R.id.text_list);

        String[] strings = {"one", "two", "three", "four", "five", "sex", "seven", "eight", "nine", "ten"};

        text_list.setText("");
        for (String x : strings)
            text_list.append(x + "\n");
    }
}

File: Kotlin/MyMovies/app/src/main/java/com/example/android/mymovies/activity/MainActivity.kt
package com.example.android.mymovies.activity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.lifecycle.Observer
import androidx.lifecycle.ViewModelProvider
import androidx.recyclerview.widget.GridLayoutManager
import com.example.android.mymovies.MovieViewModel
import com.example.android.mymovies.R
import com.example.android.mymovies.databinding.ActivityMainBinding
import com.example.android.mymovies.pogo.Movie
import com.example.android.mymovies.recyclerView.MovieAdapter

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding
    private lateinit var movieViewModel: MovieViewModel
    private lateinit var movieAdapter: MovieAdapter
    private lateinit var movies: List<Movie>

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        movies = ArrayList()
        movieAdapter = MovieAdapter(movies as ArrayList<Movie>)

        movieViewModel = ViewModelProvider(this).get(MovieViewModel::class.java)
        movieViewModel.fetchMovies()
        movieViewModel.popularMoviesLiveData.observe(this, Observer {
            movieAdapter.addMovies(it)
        })

        binding.recyclerViewPosters.layoutManager = GridLayoutManager(this, 2)
        binding.recyclerViewPosters.adapter = movieAdapter
    }


}
